packages/UI/src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,217 @@
+      disclaimers or limitations of liability outlined therein. Your continued
+      use of the service constitutes acceptance of these terms.
+    </p>
+  </div>
+);
+const ToSFooter = (
+  <div className="flex h-10 w-full items-center justify-between font-nunito-sans">
+    <div className="flex items-center justify-center">
+      <Checkbox />
+      <p className="ml-2">I've read the terms of services</p>
+    </div>
+    <Close>
+      <Button buttonType="primary" buttonSize="standard">
+        Submit
+      </Button>
+    </Close>
+  </div>
+);
+
+const PrimaryTriggerBtn = (
+  <Button buttonType="primary" buttonSize="standard">
+    Subscribe!
+  </Button>
+);
+const PrimaryHeader = (
+  <h3 className="w-full font-nunito-sans text-lg uppercase tracking-wider">
+    {' '}
+    Subscribe to our newsletter{' '}
+  </h3>
+);
+const PrimaryContent = (
+  <div className="">
+    <p className="w-96 pl-8 font-nunito-sans">
+      {' '}
+      <Description>
+        Subscribe to our newsletter for the latest trends and exclusive content
+        delivered straight to your inbox!
+      </Description>
+    </p>
+    {/* Email address label with an email address input element */}
+    <div className="flex flex-col items-center justify-center">
+      <label className="font-nunito-sans text-sm font-bold uppercase tracking-wide">
+        {' '}
+        Email address{' '}
+      </label>
+      <input
+        className="w-96 rounded-lg border-2 border-solid border-black p-2"
+        type="email"
+        placeholder="Enter your email address"
+      />
+    </div>
+  </div>
+);
+const PrimaryFooter = (
+  <div className="flex items-center justify-end gap-6">
+    <Close>
+      <Button buttonType="primary" buttonSize="standard">
+        {' '}
+        dismiss{' '}
+      </Button>
+    </Close>
+    <Close>
+      <Button buttonType="primary" buttonSize="standard">
+        {' '}
+        Subscribe{' '}
+      </Button>
+    </Close>
+  </div>
+);
+
+export const Primary: Story = {
+  render: () => (
+    <div className="flex gap-4">
+      <Modal
+        trigger={PrimaryTriggerBtn}
+        header={PrimaryHeader}
+        content={PrimaryContent}
+        footer={PrimaryFooter}
+      />
+      <Modal
+        trigger={ToSTriggerBtn}
+        header={ToSHeader}
+        content={ToSContent}
+        footer={ToSFooter}
+      />
+      <Modal
+        trigger={RoleTriggerBtn}
+        header={RoleHeader}
+        content={RoleContent}
+        footer={RoleFooter}
+      />
+      <Modal
+        trigger={EmptyTriggerBtn}
+        header={EmptyHeader}
+        content={EmptyContent}
+        footer={EmptyFooter}
+      />
+    </div>
+  ),
+};
+
+export const PrimaryNoCloseBtn: Story = {
+  render: () => (
+    <Modal
+      trigger={PrimaryTriggerBtn}
+      header={PrimaryHeader}
+      content={PrimaryContent}
+      footer={PrimaryFooter}
+      hasCloseBtn={false}
+    />
+  ),
+};
```

Comment:
the code to shown to use these stories doesn't make alot of sense, maybe we should find a solution to make it show jsx rather than `PrimaryTriggerBtn` for example
![image](https://github.com/dev-launchers/dev-launchers-platform/assets/27841994/3c3a2a86-d80d-4e61-bfef-e2809bb5e96c)

Reply:
![image](https://github.com/dev-launchers/dev-launchers-platform/assets/109195898/7e649885-6053-42e5-89c1-46abec56fca1)